### PR TITLE
[MODULAR] Reduces the Blueshield and NT Consultant's IDs to Silver to reduce the number of all-access-compatible IDs on the station back down to the intended 2

### PR DESCRIPTION
**Pull request by Iamgoofball**
## About The Pull Request

Reduces the Blueshield and NT Consultant's IDs to Silver to reduce the number of all-access-compatible IDs on the station back down to the intended 2

## How This Contributes To The Skyrat Roleplay Experience

Please consult the design document for the ID card trim system as designed at https://hackmd.io/@tgstation/HkS-z64NP#High-Access-Jobs-and-Low-Pop-Access and all relevant discussion on https://github.com/tgstation/tgstation/pull/56910 as to why all access compatible ID cards should only be available in 2 possible locations at maximum without admin intervention via events or spawning people with higher-tier cards, one on a spawned Captain and the other on the spare golden ID card in the spare safe.

Pairs well with https://github.com/Skyrat-SS13/Skyrat-tg/pull/23683 but can be merged without.

## Proof of Testing

This is battle tested code.

## Changelog
:cl:
balance: Reduces the Blueshield and NT Consultant's IDs to Silver to reduce the number of all-access-compatible IDs on the station back down to the intended 2
/:cl:

### CODE REVIEW

**Comment by ghost** [comment]
Yeah this is valid.
However, "Battle Tested Code" isn't proof of testing. You have to ensure that all the accesses still work, especially to the cards' respective unique rooms.

**Comment by burgerenergy** [comment]
Keep it aesthetically Centcom, imo.

**Comment by desertmote** [comment]
Good PR but being too lazy to test and add it to the PR is a flaw.

**Comment by lessthnthree** [comment]
Yes, please have their cards use the blue icon_state until it's decided what's going on with ID cards going forward.

**Comment by Iamgoofball** [comment]
I am not doing that, that defeats the purpose of the ID cards being recognizable at a glance when it comes to what access the ID card can hold. We can make the back-color on their job icon green for central command instead to communicate that they work for Nanotrasen.

**Comment by Iamgoofball** [comment]
stale b gone

**Review by lessthnthree — APPROVED** [review]

File: modular_skyrat/modules/nanotrasen_rep/code/nanotrasen_consultant.dm
Comment (lessthnthree) [inline modular_skyrat/modules/nanotrasen_rep/code/nanotrasen_consultant.dm:73]: This doesn't work.

**Review by lessthnthree — CHANGES_REQUESTED** [review]

**Comment by lessthnthree** [comment]
## WARNING: Unable to apply trim /datum/id_trim/job/nanotrasen_consultant to the silver identification card () in outfit Nanotrasen Consultant. in code/datums/outfit.dm at line 208 src: Nanotrasen Consultant usr: LT3.

Battle tested code my ass.

**Comment by lessthnthree** [comment]
> I am not doing that, that defeats the purpose of the ID cards being recognizable at a glance when it comes to what access the ID card can hold. We can make the back-color on their job icon green for central command instead to communicate that they work for Nanotrasen.

The request is for a unique looking ID. Until that requirement is met this won't be merged.

File: modular_skyrat/modules/nanotrasen_rep/code/nanotrasen_consultant.dm
Comment (Iamgoofball) [reply modular_skyrat/modules/nanotrasen_rep/code/nanotrasen_consultant.dm:73]: im gonna destroy id trim code i swear

**Comment by lessthnthree** [comment]
I'll approve this once https://github.com/tgstation/tgstation/pull/78761 hits (and you've fixed it to be working.)

Get someone to make a hologram icon for the positions

**Comment by EricZilla** [comment]
> > I am not doing that, that defeats the purpose of the ID cards being recognizable at a glance when it comes to what access the ID card can hold. We can make the back-color on their job icon green for central command instead to communicate that they work for Nanotrasen.
> 
> The request is for a unique looking ID. Until that requirement is met this won't be merged.

Yalls got my attention so
What about making a lower access varient of centcom IDs?
Helpful I hope
Also here are the new ID stuff in convenient skyrat format, I already added the new centcom ID too if yalls want it
[skyrat ID stuff.zip](https://github.com/Skyrat-SS13/Skyrat-tg/files/12803146/skyrat.ID.stuff.zip)

**Comment by lessthnthree** [comment]
> > > I am not doing that, that defeats the purpose of the ID cards being recognizable at a glance when it comes to what access the ID card can hold. We can make the back-color on their job icon green for central command instead to communicate that they work for Nanotrasen.
> > 
> > 
> > The request is for a unique looking ID. Until that requirement is met this won't be merged.
> 
> Yalls got my attention so What about making a lower access varient of centcom IDs? Helpful I hope Also here are the new ID stuff in convenient skyrat format, I already added the new centcom ID too if yalls want it [skyrat ID stuff.zip](https://github.com/Skyrat-SS13/Skyrat-tg/files/12803146/skyrat.ID.stuff.zip)

Oh, sweet. Well that solves that. Thanks much!

**Comment by SSensum13** [comment]
I find that PR stupid. What the point of having bodyguard that has 6+ heads of staff, that he can not save due not having access?

Head of staff is getting ripped appart, mangled and slowly gibbed?

- "Ples, open dor somon'e 😢 😭 "

Blueshield is not a part of sec, his goal is not to catch/kill antags, his goal is to protect the heads of staff and valuable personnel, so i see no reason for him having an AA.

**Comment by lessthnthree** [comment]
I can't tell if you're arguing for the current access or against it.

**Comment by SSensum13** [comment]
I am mad because it makes no sense. The idea of bodyguard is that he is someone who can protect you against any sorts of attacks or at least try to cover you with his body. Blueshield without access is an cuckold (sadly i have no other words to describe it). He will just look how his head of staff gets slaughtered and all what he can do - pray to gods that someone will open him a damn door.

**Comment by SSensum13** [comment]
I'd rather have Blueshield' SOPs and weapon cary policy, than having changes like that, because i rather be armed with a disabler or even unarmed, but atleast with direct access to any head of staff, than have a automatic lethal gun at any code but without access to the VIP that i must protect.

**Pull request closed without merging**
